src/vaadin-rich-text-editor.html

```diff
@@ -234,33 +234,42 @@
        *
        * The following shadow DOM parts are available for styling:
        *
-       * Part name                      | Description
-       * -------------------------------|----------------
-       * `content`                      | The content wrapper
-       * `toolbar`                      | The toolbar wrapper
-       * `toolbar-group`                | The group for toolbar controls
-       * `toolbar-button`               | The toolbar button (applies to all buttons)
-       * `toolbar-button-undo`          | The "undo" button
-       * `toolbar-button-redo`          | The "redo" button
-       * `toolbar-button-bold`          | The "bold" button
-       * `toolbar-button-italic`        | The "italic" button
-       * `toolbar-button-underline`     | The "underline" button
-       * `toolbar-button-strike`        | The "strike-through" button
-       * `toolbar-button-h1`            | The "header 1" button
-       * `toolbar-button-h2`            | The "header 2" button
-       * `toolbar-button-h3`            | The "header 3" button
-       * `toolbar-button-subscript`     | The "subscript" button
-       * `toolbar-button-superscript`   | The "superscript" button
-       * `toolbar-button-list-ordered`  | The "ordered list" button
-       * `toolbar-button-list-bullet`   | The "bullet list" button
-       * `toolbar-button-align-left`    | The "left align" button
-       * `toolbar-button-align-center`  | The "center align" button
-       * `toolbar-button-align-right`   | The "right align" button
-       * `toolbar-button-image`         | The "image" button
-       * `toolbar-button-link`          | The "link" button
-       * `toolbar-button-blockquote`    | The "blockquote" button
-       * `toolbar-button-code-block`    | The "code block" button
-       * `toolbar-button-clean`         | The "clean formatting" button
+       * Part name                            | Description
+       * -------------------------------------|----------------
+       * `content`                            | The content wrapper
+       * `toolbar`                            | The toolbar wrapper
+       * `toolbar-group`                      | The group for toolbar controls
+       * `toolbar-group-history`              | The group for histroy controls
+       * `toolbar-group-emphasis`             | The group for emphasis controls
+       * `toolbar-group-heading`              | The group for heading controls
+       * `toolbar-group-glyph-transformation` | The group for glyph transformation controls
+       * `toolbar-group-group-list`           | The group for group list controls
+       * `toolbar-group-alignment`            | The group for alignment controls
+       * `toolbar-group-rich-text`            | The group for rich text controls
+       * `toolbar-group-block`                | The group for block controls
```

Comment:
Can we slightly expand the description of this `part` as it's not so obvious at the moment what does block control mean?
(For others it's quite easy to guess by the name)

Reply:
Added improved description `The group for preformatted block controls`